types/apollo-upload-client/package.json

```diff
@@ -1,8 +1,8 @@
 {
     "private": true,
     "dependencies": {
-        "apollo-link": "^1.2.12",
-        "apollo-link-http-common": "^0.2.4",
-        "graphql": "^14.5.3"
+        "@apollo/client": "^3.0.0",
+        "csstype": "^3.0.2",
```

Comment:
I think this will fail because csstype isn't on the list of approved deps ? I know this is trying to work around the linting error, but I don't think this will ever work